### DISCUSSION TITLE
Do mobile users miss out on server announcements?

### DISCUSSION THREAD

**Discussion by FLPresets** [mastodon/mastodon#23080] [Q&A]
I don't see where mobile users can see server announcements.  

Surely they must- what good are announcements if half the users don't see them?  But I'm not getting notifications on mobile for server announcements.  I don't even see a place to find them at all.

Where do I find server announcements on the mobile app(s)?  Thanks

**Comment by rbairwell** [comment]
Which mobile app are you using on what type of phone?

I can confirm that - through experience - that the Official [Mastodon app for Android](https://app.joinmastodon.org/android) does NOT currently have Announcement support (this is a known issue https://github.com/mastodon/mastodon-android/issues/82 ). But I can see announcements in the sidebar in [Tusky ](https://tusky.app/), same location for [Fedilab ](https://fedilab.app/), and shown in login on [Tooot!](https://tooot.app/) . I can't see announcements in [Megaladon](https://sk22.github.io/megalodon/) ( again, known issue https://github.com/sk22/megalodon/issues/127 )

I believe that the Official [Mastodon app for iOS ](https://app.joinmastodon.org/ios) does support announcements : but I do not have an Apple device to test it or the other apps on.
